__tests__/__snapshots__/storyshots.test.js.snap

```diff
@@ -16603,7 +16603,7 @@ exports[`Storyshots Pages/Signup Signup Success 1`] = `
         className="card-text"
       />
       <p>
-        You will receive a link to confirm your email address and to set your password.
+        Link to set your password: https://www.c0d3.com/confirm/undefined
```

Comment:
uh, crap, that's not good.

Reply:
It's only on the snapshot, unless you set the value there it will show as undefined. It's fine to me.

Reply:
oh I gotta update two additional files